cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx

```diff
@@ -58,15 +76,36 @@ function renderGridHeader(nodes, tableName, classes) {
   );
 }
 
-function FllTable({ tableName, fields, classes }) {
+function renderGridBody(fields, tableName, classes) {
+  return (
+    <div className={classes.gridBody} id={`${tableName}`}>
+      {fields.map((field) => {
+        return (
+          <div
+            className={classnames('grid-row', {
+              'grid-link': true,
```

Comment:
is this `grid-link` will be modified later to evaluate on some expression? if it's always true, it should just be set on the className directly